addons/analytic/models/analytic_plan.py

```diff
@@ -174,6 +174,13 @@ def _compute_children_count(self):
         for plan in self:
             plan.children_count = len(plan.children_ids)
 
+    @api.onchange('parent_id')
+    def _onchange_parent_id(self):
+        project_plan, __ = self._get_all_plans()
+        for plan in self:
+            if plan.id.origin == project_plan.id:
```

Comment:
are you sure `.id.origin` works? usually I'd see `._origin.id` instead :thinking:

Reply:
the id is <NewId> object used in onchanges. You can see another example in mrp
https://github.com/odoo/odoo/blob/98d9f02f89b9f04c9d2d3bf0ba73c4778d511c4b/addons/mrp/models/mrp_bom.py#L228